src/containers/HabitatProject/HabitatProjectContent.tsx

```diff
@@ -50,6 +50,107 @@ const Responsibilities = (props: { children: any; }) => {
   );
 };
 
+
+const AboutProject = () => {
```

Comment:
@kathleenahall I don't think you meant to make these changes on this page, please put it back how it was before. I think you messed up when you were fixing merge conflicts, thank you

Reply:
This does look like something that happened when I tried to fix the merge conflicts in Github, but these changes are not showing up in vs code so I'll push everything up once I fix the merge conflicts in vscode.